lib/screens/forgotpassword/forgotpassword.screen.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import '../signup/signup.screen.dart';
+
+class ForgotPassword extends StatefulWidget {
+  const ForgotPassword({super.key});
+
+  @override
+  State<ForgotPassword> createState() => _ForgotPasswordState();
+}
+
+class _ForgotPasswordState extends State<ForgotPassword> {
+  String email="";
+  TextEditingController _emailController = new TextEditingController();
+  final _formkey = GlobalKey<FormState>();
+
+  resetPassword() async {
+    try {
+      await FirebaseAuth.instance.sendPasswordResetEmail(email: email);
+      ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+          content: Text(
+            "Password Reset Email has been sent !",
+            style: TextStyle(fontSize: 20.0),
+          )));
+    } on FirebaseAuthException catch (e) {
+      if (e.code == "user-not-found") {
+        ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+            content: Text(
+              "No user found for that email.",
+              style: TextStyle(fontSize: 20.0),
+            )));
+      }
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        body: Container(
+          decoration: const BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage("assets/SL-051919-20420-09.jpg"),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Container(
+            margin: const EdgeInsets.all(24),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                _header(context),
+                _inputField(context),
+                _signup(context),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  _header(BuildContext context) {
+    return Row(
+      children: [
+        Image.asset(
+          "assets/logo.png", // Replace with your image asset path
+          height: 90, // Adjust the height as needed
+        ),
+        const SizedBox(width: 10), // Add some space between the logo and text
+        Expanded(
+          child: LayoutBuilder(
+            builder: (context, constraints) {
+              final maxWidth = constraints.maxWidth;
+              final textStyle = TextStyle(
+                fontSize: 30.0,
+                fontWeight: FontWeight.bold,
+                color: Color(0xFF36A383),
+              );
+
+              final welcomeText = Text(
```

Comment:
textul `welcomeText` si `recycleCashText` au exact acelasi style
poti face o functie separata pt ele gen textInHeader() si sa ii dati ca parametru textul. Daca faci asta, pe langa ca nu scrieti aceeasi bucata de cod de 2 ori, curatati functia asta, si nici nu creati variabile pt texte ;) 

si atunci ar fi asa:
```
Widget textInHeader( String text, double maxWidth ) {  // aici nu stiu daca maxWidth e int sau double
  final textStyle = TextStyle(
      fontSize: maxWidth / 8,
      fontWeight: FontWeight.bold,
      color: Color(0xFF36A383),
    );

    return Text(
        text,
        style: textStyle,
        overflow: TextOverflow.ellipsis,
}
```

si in  _header() apelati asa:

```
...
return Column(
    ...
    children: [
        textInHeader("Welcome to", maxWidth ),
        textInHeader("RecycleCash")
    ]
);

```

hope it makes sense